content/en/docs/concepts/secure-deployment.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Securely deploying Notation"
+description: "Best practices for securely deploying Notation"
+type: docs
+weight: 2
+---
+
+A deployment of Notation, specifically the `notation` CLI, is used for signing and verifying container images. The `notation` CLI is usually deployed on a development computer or using a CI/CD pipeline, such as GitHub Actions.
+
+A deployment has several sensitive components:
+
+- Security policies
```

Comment:
Maybe we should link to the documentation for those

Reply:
fixed